src/routes/[deviceUID]/dashboard/+page.svelte

```diff
@@ -54,7 +54,7 @@
   let eventsUrl = `https://notehub.io/project/${APP_UID}/events`;
 
   $: if (deviceUID) {
-    eventsUrl = `https://notehub.io/project/${APP_UID}/events?queryDevice=${deviceUID}`;
+    eventsUrl = `https://notehub.io/project/${APP_UID}/events?filter_dev=${deviceUID}`;
```

Comment:
This same URL is in here twice. Maybe worth moving to a utility function?

Reply:
Good suggestion. I'll put a function in the url.ts helper file